locales/de/common.json

```diff
@@ -169,40 +169,40 @@
   "{{ count }} year ago_plural": "Vor {{ count }} Jahren",
   "Log out": "Abmelden",
   "Functions": "Funktionen",
-  "One or more files have been modified on disk since Vortex last deployed. You have to decide what happens with them.": "Eine oder mehrere Dateien wurden verändert seit Vortex das letzte mal deployed hat. Sie müssen wählen was mit diesen Dateien passieren soll.",
-  "Cancel deployment": "Deployment abbrechen",
+  "One or more files have been modified on disk since Vortex last deployed. You have to decide what happens with them.": "Eine oder mehrere Dateien wurden seit der letzten Mod-Bereitstellung verändert. Sie müssen entscheiden, was mit diesen Dateien passieren soll.",
+  "Cancel deployment": "Bereitstellung abbrechen",
```

Comment:
Auch hier gilt: Zu viel zu übersetzen kann mehr Verwirrung stiften als es hilft. "Deployment" ist der Begriff den wir in Foren-Posts, Doku usw. verwenden. Es ist gut für die Nutzer wenn sie die englischen Begriffe kennen damit sie ggfs. nach Hilfe suchen können.
Wenn wir Deployment übersetzen müsste es konsistent überall mit "Bereitstellung" übersetzt werden und es muss jemand dafür sorgen dass auf deutsch ebenfalls Doku existiert und aktuell gehalten wird die diese Begrifflichkeiten verwendet. Der Aufwand wäre enorm.